script.js

```diff
@@ -0,0 +1,181 @@
+const menu = document.querySelector('.hamburger');
+const desktopnav = document.querySelector('.desktop-nav');
+// ............ Adding click event to the hamburger.........................
+menu.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  desktopnav.classList.toggle('active');
+});
+document.querySelectorAll('.ham-item').forEach((n) => n.addEventListener('click', () => {
+  menu.classList.remove('active');
+  desktopnav.classList.remove('active');
+}));
+// ............Adding event to close icon on menu.............................
+document.querySelector('.close-icon').addEventListener('click', () => {
+  menu.classList.remove('active');
+  desktopnav.classList.remove('active');
+});
+
+// Objects
+const projectContainerArr = [
```

Comment:
- [ ] Your featured speakers section is not displaying for now please make it visible 👍 

![image](https://user-images.githubusercontent.com/75126481/177012838-eb35d64f-8575-46ea-98db-1bfd7f332324.png)